pyramid_okta/security/authentication.py

```diff
@@ -65,6 +64,7 @@ def effective_principals(self, request):
             effective_principals.append(user_id)
         else:
             effective_principals.append(Authenticated)
+            effective_principals.append('login_required')
```

Comment:
....wait.  i can't even find the documentation on `'login_required'`...is that something I made up?  If so...we shouldn't support it here...